web/pgadmin/dashboard/static/js/SystemStats/Memory.jsx

```diff
@@ -66,29 +67,29 @@ export default function Memory({preferences, sid, did, pageVisible, enablePoll=t
 
   const tableHeader = [
     {
-      Header: 'PID',
-      accessor: 'pid',
+      Header: gettext('PID'),
+      accessor: gettext('pid'),
```

Comment:
accessor should not be in gettext, it is used internally. gettext is basically used for language translation, so whatever text is visible to the user should wrap in gettext.

Reply:
Okay understood, I'll correct it!